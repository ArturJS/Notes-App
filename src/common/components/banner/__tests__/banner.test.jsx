import React from 'react';
import renderer from 'react-test-renderer';
import Banner from '../banner';

describe('<Banner />', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Banner title="test title" body="test body" />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    });
});
