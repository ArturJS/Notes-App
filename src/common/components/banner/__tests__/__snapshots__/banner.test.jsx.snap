// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Banner /> renders correctly 1`] = `
<div
  className="banner"
>
  <h3
    className="banner__title"
  >
    test title
  </h3>
  <p
    className="banner__body"
  >
    test body
  </p>
</div>
`;
